apps/website-25/src/pages/_app.tsx

```diff
@@ -2,15 +2,50 @@ import '../globals.css';
 import type { AppProps } from 'next/app';
 import Head from 'next/head';
 import dynamic from 'next/dynamic';
+import posthog from 'posthog-js';
+/* eslint-disable import/no-extraneous-dependencies */
+import { PostHogProvider } from 'posthog-js/react';
 import {
   CookieBanner, Footer, isCurrentPath, Nav, constants,
 } from '@bluedot/ui';
 import clsx from 'clsx';
+import { useEffect } from 'react';
+import { Router } from 'next/router';
 import { Analytics } from '../components/Analytics';
 
+if (typeof window !== 'undefined') {
+  if (!process.env.NEXT_PUBLIC_POSTHOG_KEY) {
+    console.error('PostHog key is missing! Check your .env.local file');
+  } else {
+    console.log('Initializing PostHog...');
+    posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
+      api_host: 'https://eu.i.posthog.com',
```

Comment:
```suggestion
      api_host: 'https://analytics.k8s.bluedot.org',
```

(see posthog-proxy app for more details)